ext/debug/segger/Kconfig

```diff
@@ -8,9 +8,53 @@ config HAS_SEGGER_RTT
 	bool
 	default n
 
+if HAS_SEGGER_RTT
+
 config SEGGER_SYSTEMVIEW
 	bool
 	prompt "Segger SystemView support"
 	default n
-	depends on HAS_SEGGER_RTT
 	select RTT_CONSOLE
+
+config SEGGER_RTT_MAX_NUM_UP_BUFFERS
+	int "Maximum number of up-buffers"
+	default 3
+
+config SEGGER_RTT_MAX_NUM_DOWN_BUFFERS
+	int "Maximum number of down-buffers"
+	default 3
+
+config SEGGER_RTT_BUFFER_SIZE_UP
+	int "Size of the buffer for terminal output of target, up to host"
+	default 1024
+
+config SEGGER_RTT_BUFFER_SIZE_DOWN
+	int "Size of the buffer for terminal input of target, from host"
+	default 16
+
+config SEGGER_RTT_PRINTF_BUFFER_SIZE
```

Comment:
Do we need this one? I don't think the console uses it?

Reply:
it's segger_rtt configuration. Other modules in zephyr may use it as well so it won't harm moving full configuration to kconfig.